src/Requests/AbstractArrayParams.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Giadc\JsonApiRequest\Requests;
+
+abstract class AbstractArrayParams implements RequestInterface
+{
+    protected array $container = [];
+
+    public function __construct(?array $items = [])
+    {
+        foreach ($items as $key => $item) {
+            $this->add($key, $item);
+        }
+    }
+
+    abstract protected static function paramKey(): string;
+
+    /**
+     * Add an additional item.
+     *
+     * @param string|array $value
+     */
+    public function add(string $key, $value): void
+    {
+        if (is_string($value)) {
+            $value = explode(',', $value);
+        }
```

Comment:
Annoying PHP quirk: If `$value` is an empty string `explode(',', $value)` results in `[""]`. That is, a one-item array with a single, empty-string value.


```suggestion
        if ($value == '') {
            return [];
        }
        
        if (is_string($value)) {
            $value = explode(',', $value);
        }
```
We would expect it to simply be an empty array (`[]`).